bareos/traymonitor.sls

```diff
@@ -31,5 +32,5 @@ bareos_traymonitor:
     - user: root
     - group: root
     - require:
-      - pkg: bareos_traymonitor
+      - pkg: bareos-traymonitor
```

Comment:
Same as with the other deps, can you change it to depend on `install_traymon_package`?